include/mrdox/Platform.hpp

```diff
@@ -19,6 +21,15 @@
 namespace clang {
 namespace mrdox {
 
+//------------------------------------------------
+//
+// Check llvm version
+//
+//------------------------------------------------
+
+static_assert(LLVM_VERSION_MAJOR >= 15,
```

Comment:
Put this in Platform.hpp (it doesn't require llvm includes)
```
#define MRDOX_MINIMUM_LLVM_VERSION 15
```

And then use that in Config.cpp to do the do